### DISCUSSION TITLE
Different markdown formatting

### DISCUSSION THREAD

**Discussion by ghost** [vincode-io/Zavala#95] [Ideas]
I thought about md export - right now the output is formatted as list, but what if I want to process later in md editor and publish on blog? I have to change every line to headline, if Zavala could export level 1 as H1, level 2 as H2 it would be great. It's small detail, but very convenient as from few days all my texts starts in Zavala.

**Comment by vincode-io** [comment]
Interestingly, I had that as a 1.0 feature, but pulled it at the last minute. I wasn't sure it was a good fit or not. I am adding for version 1.1 a Jekyll export option. I was debating about hiding that behind a hidden preference [the way that NetNewsWire](https://github.com/Ranchero-Software/NetNewsWire/blob/main/Technotes/HiddenPrefs.md) does it. Maybe if enough people would use it, it could just be left available all the time.

Which markdown editor do you use and which blogging platform do you use? I might need to create two separate exports. One that does images and one that ignores them.

**Reply by ghost** [reply]
iAWriter and Wordpress. Can test something if needed. 

Have you thought about importing md? The same as above but in opposite direction - start as headlines in md editor and continue in Zavala. It could be a little overwhelming, but also could provide nice workflow between md editor and Zavala's outlines. I have a slight suspicion that Zavala may become popular with writers.

**Reply by vincode-io** [reply]
Importing markdown into an outline is crazy hard. There are just so many edge cases. I'm not going to totally rule it out, but it won't be happening anytime soon.

For now the workflow will have to be Zavala, then editor, then publish. I would love to be able to round trip the workflow. We'll see what happens.

**Reply by ghost** [reply]
Fair enough. I’m also thinking that Zavala -> editor -> publishing is the most popular direction.

**Reply by vincode-io** [reply]
Too bad iA Writer doesn't support the TextBundle format. It would be the perfect way to interoperate with it and Zavala. I will probably go ahead and implement a TextBundle export some day so that Ulysses users can at least take advantage of it.
